Task 2/Task 2.1.2/Task 2.1.2/Program.cs

```diff
@@ -0,0 +1,804 @@
+        {
+            return $"Квадрат.\n" +
+                   $"Координаты вершин квадрата - (x:, y:) {StartpointX} {StartpointY}, {StartpointX + Side} {StartpointY}, {StartpointX + Side} {StartpointY + Side}, {StartpointX} {StartpointY + Side}\n" +
+                   $"площадь квадрата - {Math.Round(GetSquare, 2, MidpointRounding.AwayFromZero)}\n" +
+                   $"периметр квадрата - {Math.Round(GetPerimeter, 2, MidpointRounding.AwayFromZero)}";
+        }
+    }
+
+    class Rectangle : Quadrate
+    {
+        public Rectangle(float pointX, float pointY, float side, float height) : base (pointX, pointY, side)
+        {
+            StartpointX = pointX;
+            StartpointY = pointY;
+            Side = side;
+            Height = height;
+        }
+
+        private float _height;
+
+        public float Height
+        {
+            get => _height;
+            set => _height = value;
+        }
+
+        public override double GetSquare
+        {
+            get
+            {
+                return Side * Height;
+            }
+        }
+
+        public override double GetPerimeter
+        {
+            get
+            {
+                return (Side + Height) * 2;
+            }
+        }
+
+        public override string ToString()
+        {
+            return $"Прямоугольник.\n" +
+                   $"Координаты вершины прямоугольника - (x:, y:) {StartpointX} {StartpointY}, {StartpointX + Side} {StartpointY}, {StartpointX + Side} {StartpointY + Height}, {StartpointX} {StartpointY + Height}\n" +
+                   $"площадь прямоугольника - {Math.Round(GetSquare, 2, MidpointRounding.AwayFromZero)}\n" +
+                   $"периметр прямоугольника - {Math.Round(GetPerimeter, 2, MidpointRounding.AwayFromZero)}";
+        }
+    } 
+
+    class Sphere : VolumeFigures
+    {
+        public Sphere(float pointX, float pointY, float radius)
+        {
+            StartpointX = pointX;
+            StartpointY = pointY;
+            Radius = radius;
+        }
+
+        private float _radius;
+        public float Radius
+        {
+            get => _radius;
+            set => _radius = value;
+        }
+
+        public override double GetSquare
+        {
+            get
+            {
+                return 4 * Math.PI * Radius * Radius;
+            }
+        }
+
+        public override double GetVolume
+        {
+            get
+            {
+                return (4 * Math.PI * Radius * Radius * Radius) / 3;
+            }
+        }
+
+        public override string ToString()
+        {
+            return $"Сфера.\n" +
+                   $"Координаты центра сферы - (x:, y:) {StartpointX} {StartpointY}\n" +
+                   $"площадь сферы - {Math.Round(GetSquare, 2, MidpointRounding.AwayFromZero)}\n" +
+                   $"объем сферы - {Math.Round(GetVolume, 2, MidpointRounding.AwayFromZero)}";
+        }
+    }
+
+    class Cylinder : Sphere
```

Comment:
Cylinder is not a case of a sphere. More like a circle that has a height in other dimension. Anyway, nobody expects a cylinder in a collection of spheres